package.json

```diff
@@ -45,7 +45,9 @@
   "module": "dist/es/src/index.js",
   "dependencies": {
     "babel-runtime": "~6.25.0",
-    "color": "^3.1.3"
+    "color": "^3.1.3",
+    "lodash": "^4.17.20",
+    "react-reconciler": "^0.26.1"
```

Comment:
Maybe should bump a version inside a package ?

Reply:
The `latest` version is `0.26.1`

Reply:
sorry, i had in mind to bump the version of `mjml-react`. It's not a breaking change, but still.